example/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.example">
+    <!-- Flutter needs it to communicate with the running application
+         to allow setting breakpoints, to provide hot reload, etc.
+    -->
+    <uses-permission android:name="android.permission.INTERNET"/>
+</manifest>
```

Comment:
Nice addition using this only in the debug `AndroidManifest.xml`

Reply:
Thanks, but give credit to the Flutter team! This change came from re-creating the example application using `flutter create`.